website/content/docs/api-gateway/install.mdx

```diff
@@ -0,0 +1,54 @@
+---
+layout: docs
+page_title: Consul API Gateway Install
+description:  >-
+  Installing Consul API Gateway
+---
+
+# Installing Consul API Gateway
+
+This topic describes how to use the Consul API Gateway add-on module. It includes instructions for installation and configuration.
+
+## Requirements
+
+Ensure that the environment you are deploying Consul API Gateway in meets the requirements listed in the [Technical Specifications][tech-specs]. This includes validating that the requirements for minimum versions of software are met. See the [Release Notes][rel-notes] for the version of API Gateway you are deploying.
+
+## Installation
+
+-> **Version reference convention:** Replace `VERSION` in command and configuration examples with the Consul API Gateway version you are installing, such as `0.3.0`. In some instances, `VERSION` is prepended with a lowercase _v_. This indicates that you must include the `v` as is part of the version, for example `v0.3.0`.
+
```

Comment:
That makes perfect sense and I'm pretty sure @mikemorris started moving our docs in the direction of using an environment variable.

Reply:
Yea, I reworked this section in https://github.com/hashicorp/consul/pull/13681 - had been intending to wait until after this restructure was merged, but looks like my PR got merged early, this could be rebased to pick up that change.